home-data/src/main/resources/application.properties

```diff
@@ -4,4 +4,4 @@ spring.datasource.password=${DATASOURCE_PASSWORD:password}
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.jersey.type=filter
 server.port=${PORT:8080}
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=validate
```

Comment:
We should leave a blank line at the end

Reply:
Ok

Reply:
Do we need to use spring.jpa.hibernate.ddl-auto at all?

Reply:
I think yes. Why not?

Reply:
Yeah, figured it out. It can be here